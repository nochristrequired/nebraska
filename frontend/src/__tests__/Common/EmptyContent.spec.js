import { render } from '@testing-library/react';
import React from 'react';
import Empty from '../../components/Common/EmptyContent';

describe('Empty', () => {
  it('renders correct content', () => {
    const { asFragment } = render(<Empty>Some Dummy Content</Empty>);
    expect(asFragment()).toMatchSnapshot();
  });
});
