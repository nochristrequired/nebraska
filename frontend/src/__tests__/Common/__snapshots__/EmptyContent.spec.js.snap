// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty renders correct content 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
      data-testid="empty"
    >
      Some Dummy Content
    </p>
  </div>
</DocumentFragment>
`;
